### ISSUE TITLE
VariantAnnotator fails to generate correct reference-based annotations

### ISSUE THREAD

**Issue by tfenne** [broadinstitute/gatk#6095]
### Affected tool(s) or class(es)
VariantAnnotator -A ReferenceBases -A TandemRepeat

### Affected version(s)
- [x] Latest public release version [4.1.3]
- [ ] Latest master branch as of [date of test?]

### Description 
VariantAnnotator doesn't seem to correctly generate annotations that are based off the reference.  I've tried on a few different VCFs and with different genome builds, and get the same result every time.

For `ReferenceBases` it seems to generate a string for each variant that is the ref allele, followed by `20-len(ref) * N`.  E.g.:

```
1       118617  rs372912307     T       C       50      PASS    REF_BASES=TNNNNNNNNNNNNNNNNNNNN
1       567239  rs78150957      CG      C       50      PASS    REF_BASES=CGNNNNNNNNNNNNNNNNNNN
```

The STR annotations get their header lines added to the header, but not a single variant is flagged as an STR.  I've tried processing the GIAB VCFs for NA12878 and NA24385 with the same results - even obvious STR variants are not flagged.  I suspect this is related to the fact that REF_BASES isn't compute properly.

#### Steps to reproduce
Take any decent size VCF without the above annotations (e.g. GIAB VCFs) and run something like:

gatk VariantAnnotator -V NA12878.vcf.gz -O NA12878.ann.vcf.gz -A TandemRepeat -A ReferenceBases -R hg38.fa

**Comment by droazen** [comment]
@takutosato It looks like you may have modified the `ReferenceBases` annotation in https://github.com/broadinstitute/gatk/pull/5151 to pad with N's -- mind commenting on this ticket? Thanks!

**Comment by takutosato** [comment]
@tfenne @droazen I was able to reproduce the bug using a jar from last October, but it seems to be fixed in the latest release (4.1.4.1). @tfenne could you try using the most recent release?

**Comment by takutosato** [comment]
Fixed as of Version 4.1.4.1
